File: 111術科模擬試題/A2.cpp
/*
#111商業技藝競賽模擬試題
#Problem #A2 MOD
#Author: Yotrew Wing
#2022/10/17
#https://github.com/yotrew/commercial_skill_competition
*/
#include <iostream>

using namespace std;

int main()
{
    int x,y,t;
    cin>>x>>y;
    if(x>y){
        t=y;
        y=x;
        x=t;
    }
    //cout<<x<<":"<<y;
    for(int i=x+1;i<y;i++){
        if(i%5==2 || i%5==3)
            cout<<i<<endl;
    }

    return 0;
}
/*
test data1:
10
18
output:
12
13
17

test data2:
19
12

output:
13
17
18

*/

File: 111術科正式試題/A2.py
#111商業技藝競賽正式試題
#Problem #A2 閏年(0結束)
#110商業技藝競賽模擬試題
#Problem B4 閏年
#2022/12/04
#https://github.com/yotrew/commercial_skill_competition

while True:
    year=int(input())
    if(year==0):
        break
    if year % 4 != 0:
        print("a normal year")
    elif year% 400 ==0:
        print("a leap year")
    elif year% 4 ==0 and year% 100 !=0:
        print("a leap year")
    else:
        print("a normal year")
'''
Sample Input 1
1992
1991
1700
2400
0
Sample Output 1
a leap year
a normal year
a normal year
a leap year
'''
